### ISSUE TITLE
Found 2 errors.

### ISSUE THREAD

**Issue by buonan** [jeremyben/json-server-auth#4]
When I build on linux I get these errors any ideas?

$git clone https://github.com/jeremyben/json-server-auth.git
$cd json-server-auth/

$ rimraf dist && tsc -p tsconfig.build.json
src/bin.ts:62:4 - error TS2352: Conversion of type 'undefined' to type 'string[]' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.

62  ;(<string[]>argv.middlewares) = [__dirname]
      ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/bin.ts:69:39 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string | number | Buffer | URL'.
  Type 'unknown' is not assignable to type 'URL'.

69  let routes = JSON.parse(readFileSync(argv.routes, 'utf8'))
                                         ~~~~~~~~~~~


Found 2 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

This is my configuration.
$ tsc --version
Version 3.4.5
$ node --version
v8.12.0
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial

**Comment by jeremyben** [comment]
Thanks, working on it. It's due to a more recent version of typescript.

**Comment by jeremyben** [comment]
Fixed in v1.2.0
